printer_zpl2/lib/zpl2.py

```diff
@@ -0,0 +1,335 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016 SYLEAM (<http://www.syleam.fr>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+class Zpl2(object):
```

Comment:
Ping me when you do and I'll be happy to review it. I recommend maybe also taking a look at a [helper library I've been playing with that's fairly similar to MQT, but for py-libs](https://github.com/LasLabs/python-quality-tools).